docs/make.jl

```diff
@@ -1,13 +1,18 @@
+using ApproximateGPs
+
 ### Process examples
 # Always rerun examples
 const EXAMPLES_OUT = joinpath(@__DIR__, "src", "examples")
 ispath(EXAMPLES_OUT) && rm(EXAMPLES_OUT; recursive=true)
 mkpath(EXAMPLES_OUT)
 
+# Obtain path of ApproximateGPs so we can run examples with the same version
+const PATH_ApproximateGPs = dirname(dirname(pathof(ApproximateGPs)))
```

Comment:
It's a bit confusing what "path of ApproximateGPs" is. `pathof` returns $REPOROOT/src/ApproximateGPs.jl; it takes a bit of processing to realise that the dirname is essentially a `/..`, and then `PATH_ApproximateGPs` is $REPOROOT.

The constant would be a bit more self-explanatory/less ambiguous with a name such as `REPOROOT` or something like that.
```suggestion
const REPOROOT = dirname(dirname(pathof(ApproximateGPs)))  # pathof() is :/src/ApproximateGPs.jl, we want parent's parent
```

Reply:
It's not necessarily the root of the repo - it's the path to the ApproximateGPs package that is used to build the docs and loaded above. You can build the docs with whatever version of ApproximateGPs you want - it doesn't have to be the checked out version (if you downloaded the whole repo at all). And the path will always point to the currently used version and ensure that the examples use the same. Therefore I didn't call it repo root on purpose.

Reply:
how about `PACKAGE_ROOT` then?

Reply:
:+1: for `pkgdir` - I think that makes the whole discussion redundant as we can just call `"$(pkgdir(ApproximateGPs))"` directly instead of needing to store it in a constant.